seq2science/rules/qc.smk

```diff
@@ -693,7 +693,8 @@ def get_alignment_qc(sample):
     # add samtools stats
     output.append(f"{{qc_dir}}/markdup/{{{{assembly}}}}-{sample}.samtools-coordinate.metrics.txt")
     output.append(f"{{qc_dir}}/samtools_stats/{{aligner}}/{{{{assembly}}}}-{sample}.samtools-coordinate.samtools_stats.txt")
-    if sieve_bam(config):
+    if sieve_bam(config) and \
+            not (get_workflow() == "rna_seq" and config.get('quantifier') == 'salmon'):
```

Comment:
Is this correct?

Reply:
Yep! It will add post-sieve data, unless Salmon is used for quantification. 

Why: if Salmon is used, the sieving does not effect the expression values, so adding it to the MultiQC is confusing (other quantifiers do use the sieved bams).

Reply:
Ah ik I understand. Maybe add a comment about it? 😇